website/cue/reference/releases/0.24.0.cue

```diff
@@ -0,0 +1,715 @@
+			scopes: ["sample transform"]
+			description: """
+				The `sample` transform can now sample trace events.
+				"""
+			pr_numbers: [13610]
+		},
+		{
+			type: "enhancement"
+			scopes: ["sources"]
+			description: """
+				A new
+				[`gcp_chronicle_unstructured`](/docs/reference/configuration/sinks/gcp_chronicle_unstructured/)
+				sink was added to send log events to [GCP
+				Chronicle](https://cloud.google.com/chronicle/docs/overview) as unstructured
+				events. We expect to support UDM events in the future.
+				"""
+			pr_numbers: [13550]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vrl", "vrl: compiler"]
+			breaking: true
+			description: """
+				A number of VRL type definition bugs have been resolved to allow VRL to more
+				precisely know the types of values and fields in more places. In general, this means
+				less need for type coercion functions like `string()`.
+
+				See [the release
+				highlight](/highlights/2022-08-16-0-24-0-upgrade-guide#vrl-type-def) for more
+				details.
+				"""
+			pr_numbers: [13619]
+		},
+		{
+			type: "fix"
+			scopes: ["enrichment tables"]
+			description: """
+				Vector no longer panics after reloading an existing enrichment table that had index
+				updates.
+				"""
+			pr_numbers: [13704]
+		},
+		{
+			type: "enhancement"
+			scopes: ["host_metrics source"]
+			description: """
+				The `host_metrics` source now emits `mode=iowait` for `host_cpu_seconds_total`.
+				"""
+			contributors: ["charmitro"]
+			pr_numbers: [13696]
+		},
+		{
+			type: "chore"
+			scopes: ["vrl", "vrl: stdlib"]
+			description: """
+				VRL's `parse_grok` and `parse_groks` implementation was unified to avoid
+				inconsistencies in pattern handling between the two.
+				"""
+			pr_numbers: [13635]
+		},
+		{
+			type: "fix"
+			scopes: ["host_metrics source"]
+			description: """
+				The `host_metrics` source has improved handling of fetching cgroups metrics from
+				hybrid cgroups. It now checks all possible locations for stats.
+				"""
+			pr_numbers: [13719]
+		},
+		{
+			type: "enhancement"
+			scopes: ["prometheus_remote_write sink", "papertrail sink", "prometheus_exporter sink", "socket sink", "pulsar sink"]
+			description: """
+				End-to-end acknowledgement support has been added to the following sinks:
+
+				- `papertrail`
+				- `socket`
+				- `pulsar`
+				- `prometheus_exporter`
+				- `prometheus_remote_write`
+				"""
+			pr_numbers: [13741, 13744, 13750, 13755]
+		},
+		{
+			type: "feat"
+			scopes: ["apex sink"]
+			description: """
+				A new [`apex` sink](/docs/reference/configuration/sinks/apex/) was added to send
+				logs to [Apex](https://apex.sh/logs/).
+				"""
+			contributors: ["mcasper"]
+			pr_numbers: [13436]
+		},
+		{
+			type: "enhancement"
+			scopes: ["aws provider"]
+			description: """
+				AWS components now allow specifying a different region to use when assuming role
+				than the region otherwise configured on the component via `auth.region`.
```

Comment:
The bit "when assuming role than" reads weird for me but maybe that is just me not knowing much about AWS.

Reply:
Yeah, it is a little awkward. I'll reword.